src/test/java/uk/ac/cam/cl/dtg/segue/api/managers/UserManagerTest.java

```diff
@@ -216,16 +217,15 @@ void getCurrentUser_IsAuthenticatedWithValidHMAC_userIsReturned()
         .andReturn(ImmutableMap.of(returnUser, false)).atLeastOnce();
     replay(dummyQuestionDatabase);
 
-    expect(dummyMapper.map(returnUser, RegisteredUserDTO.class)).andReturn(new RegisteredUserDTO()).atLeastOnce();
```

Comment:
I've noticed that in the tests we now use real mappers instead of mocks, so all these assertions are now being lost. Why was it done this way?

Reply:
I... can't actually remember why that was changed? I assume there must have been some issue with the tests while I was swapping the libraries over but what that was I'm not sure...
I've switched the affected unit tests back to mocks and they seem to be passing fine now at least...